dbt_subprojects/solana/models/_sector/dex/dex_solana_trades.sql

```diff
@@ -34,9 +34,9 @@ SELECT bt.blockchain
       , bt.trade_source
       , token_bought.symbol as token_bought_symbol
       , token_sold.symbol as token_sold_symbol
-      , case when lower(token_bought.symbol) > lower(token_sold.symbol)
-            then concat(token_bought.symbol, '-', token_sold.symbol)
-            else concat(token_sold.symbol, '-', token_bought.symbol)
+      , case when lower(COALESCE(token_bought.symbol, 'unknown')) > lower(COALESCE(token_sold.symbol, 'unknown'))
```

Comment:
this is not in line with the other dex-models and also won't help any analysis, would rather not implement this pattern.

Reply:
this also doesn't fix any problems outlined in the task this always returns the same number of rows, whether we change this to the old way, new way or just leave it out.

https://dune.com/queries/4998583
https://dune.com/queries/4998590
https://dune.com/queries/4995312

Reply:
we could consider making the null explicit, but doesn't have any great advantage as far as I can tell

```suggestion
      , case when token_bought.symbol is null or token_sold.symbol is null then null
             when (token_bought.symbol) > lower(token_sold.symbol) then concat(token_bought.symbol, '-', token_sold.symbol)
             else concat(token_sold.symbol, '-', token_bought.symbol)
        end as token_pair
```